### ISSUE TITLE
Handle window maximization better

### ISSUE THREAD

**Issue by sarathms** [ooni/probe-desktop#21]
Currently, when the window is maximized, it creates a weird white space on the right side

**Comment by hellais** [comment]
Yes this is a good suggestion. I say we disable window maximisation and possibly also set some constraints on the maximum and minimum window size. @elioqoshi what do you think?
